README.md

```diff
@@ -39,3 +39,11 @@ ddev exec touch public/FIRST_INSTALL
 ddev launch
 ```
 
+## Remove the Patch in your composer-installed TYPO3 v11.5.x:
+
+```shell
+composer remove typo3-ux/page-module-pilot
+composer config extra.enable-patching false
```

Comment:
I would rather use `composer config --unset extra.enable-patching` here which removes the configuration. The configuration does not have any value if `cweagans/composer-patches` is not installed (and `false` is the default anyways if it is installed) so this would leave a cleaner system.